nat-lab/tests/conftest.py

```diff
@@ -363,3 +364,21 @@ def copy_file_from_container(container_name, src_path, dst_path):
         )
     except subprocess.CalledProcessError:
         print(f"Error copying log file {src_path} from {container_name} to {dst_path}")
+
+
+async def collect_mac_diagnostic_reports():
+    if not (
+        "GITLAB_CI" in os.environ or "NATLAB_COLLECT_MAC_DIAGNOSTIC_LOGS" in os.environ
```

Comment:
maybe `NATLAB_SAVE_LOGS` would be enough, instead of introducing `NATLAB_COLLECT_MAC_DIAGNOSTIC_LOGS`

Reply:
I suspect that most of the time, people running nat-lab don't have mac VM running, so I want to give them an option to not run this part and wait for some timeout to terminate diagnostic reports collection.

Reply:
:ok_hand: